Source/WebKit/UIProcess/ios/WKContentViewInteraction.mm

```diff
@@ -4237,22 +4237,15 @@ - (NSDictionary *)textStylingAtPosition:(UITextPosition *)position inDirection:(
     if (!_page->editorState().postLayoutData)
         return nil;
     auto typingAttributes = _page->editorState().postLayoutData->typingAttributes;
-    CTFontSymbolicTraits symbolicTraits = 0;
-    if (typingAttributes.contains(WebKit::TypingAttribute::Bold))
-        symbolicTraits |= kCTFontBoldTrait;
-    if (typingAttributes.contains(WebKit::TypingAttribute::Italics))
-        symbolicTraits |= kCTFontTraitItalic;
-
-    // We chose a random font family and size.
-    // What matters are the traits but the caller expects a font object
-    // in the dictionary for NSFontAttributeName.
-    RetainPtr<CTFontDescriptorRef> fontDescriptor = adoptCF(CTFontDescriptorCreateWithNameAndSize(CFSTR("Helvetica"), 10));
-    if (symbolicTraits)
-        fontDescriptor = adoptCF(CTFontDescriptorCreateCopyWithSymbolicTraits(fontDescriptor.get(), symbolicTraits, symbolicTraits));
     
-    RetainPtr<CTFontRef> font = adoptCF(CTFontCreateWithFontDescriptor(fontDescriptor.get(), 10, nullptr));
+    // Moved from old fontForCaretSelection implementation, removed because SPI
```

Comment:
Nit - not sure this comment is necessary. If we want to keep this, then I'd suggest:
```suggestion
    // Setting a font here allows us to avoid implementing the private method `-fontForCaretSelection`.
```